mw-webapp/src/component/notice/Notice.tsx

```diff
@@ -0,0 +1,62 @@
+import {useEffect} from "react";
+import {Button} from "../button/Button";
+import {Title} from "../title/Title";
+import {HeadingLevel} from "../title/Title";
+import styles from "./Notice.module.scss";
+
+const MILLISECONDS = 1000;
+
+export interface NoticeProps {
+  /**
+   * ID for notification, it will be help closing notice by button
+   */
+  id: string;
+  /**
+  * Title for notice (text)
+  */
+  title: string;
+  /**
+   * Description for notice (text)
+   */
+  description: string;
+  /**
+   * Type for notice, like 'Error', 'Ok', 'Warning' (text)
+   */
+  type: string;
+  /**
+   * Timer for automaticle closing (seconds)
+   */
+  timer: number;
```

Comment:
Do we always need this property? Maybe better make it optional. For example I need to read notification few times? then better don't close it automatically

Reply:
Sure. I'm agree with you.